CHANGELOG.rst

```diff
@@ -28,6 +28,9 @@ Python Client
 
 Added Features
 --------------
+* Added a privacy notice link to the footer which can be configure on the girder setting view (
```

Comment:
`configure on the girder setting view` => `set on the Server Configuration view of the web client`